STYLE.md

```diff
@@ -52,16 +59,26 @@ following abbreviations we assume readers will already know.
 - UTXO (Unspent Transaction Output)
 - vbyte (virtual byte)
 
+Introduce sighash with "signature hash (sighash)", but don't introduce sighash
```

Comment:
This part seems out of place to me. Should it be on its own line or as a bullet above?